tests/bigint.rs

```diff
@@ -421,6 +421,13 @@ fn test_convert_f32() {
         b <<= 1;
     }
 
+    // test correct ties-to-even rounding
+    let weird: i128 = (1i128 << 100) + (1i128 << (100 - 24));
```

Comment:
I'd rather see `MANTISSA_DIGITS` used instead of magic numbers 24 or 53 -- but that's not just you, I see the existing tests already abuse that.

Maybe we could also add tests in `ci/big_quickcheck`?

Reply:
Changed to use `MANTISSA_DIGITS` for the new tests I've added.

Added a quickcheck test. I had to make my own `Arbitrary` to increase the chance of it failing for the old rounding. A bit ugly though... What do you think?